src/config/http.rst

```diff
@@ -129,7 +129,7 @@ HTTP Server Options
             [chttpd]
             changes_timeout = 60000 ; 60 seconds
 
-    .. config:option:: config_whitelist :: Config options while list
+    .. config:option:: config_whitelist :: Config options white list
```

Comment:
I think it should also be a single word, to be consistent with the following paragraph.

```suggestion
    .. config:option:: config_whitelist :: Config options whitelist
```

Reply:
I'm also a bit surprised to see the word `whitelist` still used within CouchDB. We made an effort to remove certain words in the past, but it looks like this one didn't get caught.

Reply:
Yes, you are right. Should we rename the config option to e.g.: config_allowlist or config_acceptlist or any other better name?

Reply:
IMHO, either is fine. I'm happy to defer to anyone who has an actual preference :)

Reply:
Okay, thats a bigger discussion not only for backwards compatibility when changing config options. I will write a post on slack for further opinions, but good catch!